### PR TITLE
Fix code quality issues found by lgtm

### PR DESCRIPTION
**Pull request by mikewalch**
* Removed several cases where collections were
  created and populated but never used
* Use equals() rather than == for Long equality
* Fixed log issue

### CODE REVIEW

File: server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java [hunk 1]
```diff
@@ -123,13 +123,9 @@ private static LogEntry switchVolumes(LogEntry le, List<Pair<Path,Path>> replace
     else
       switchedPath = le.filename;
 
-    ArrayList<String> switchedLogs = new ArrayList<>();
     String switchedLog = switchVolume(le.filename, FileType.WAL, replacements);
     if (switchedLog != null) {
```
Comment (ctubbsii) [inline server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java]: This doesn't even need to be assigned to a variable... can just check if method return is null.

File: server/base/src/main/java/org/apache/accumulo/server/master/state/TabletStateChangeIterator.java [hunk 2]
```diff
@@ -267,10 +267,6 @@ public static void setMasterState(IteratorSetting cfg, MasterState state) {
 
   public static void setShuttingDown(IteratorSetting cfg, Set<TServerInstance> servers) {
     if (servers != null) {
-      List<String> serverList = new ArrayList<>();
-      for (TServerInstance server : servers) {
-        serverList.add(server.toString());
-      }
       cfg.addOption(SHUTTING_DOWN_OPTION, Joiner.on(",").join(servers));
```
Comment (ctubbsii) [inline server/base/src/main/java/org/apache/accumulo/server/master/state/TabletStateChangeIterator.java]: I guess this works because join calls toString?

Comment (mikewalch) [reply server/base/src/main/java/org/apache/accumulo/server/master/state/TabletStateChangeIterator.java]: I wrote some similar code and it looks like it does

File: server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java (hunk 1 above)
Comment (mikewalch) [reply server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java]: Fixed

File: server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKAuthorizor.java [hunk 3]
```diff
@@ -87,16 +77,7 @@ public void initializeSecurity(TCredentials itw, String rootuser)
...
-    // create the root user with all system privileges, no table privileges, and no record-level
-    // authorizations
-    Set<SystemPermission> rootPerms = new TreeSet<>();
-    for (SystemPermission p : SystemPermission.values())
-      rootPerms.add(p);
-    Map<TableId,Set<TablePermission>> tablePerms = new HashMap<>();
-    // Allow the root user to flush the metadata tables
-    tablePerms.put(MetadataTable.ID, Collections.singleton(TablePermission.ALTER_TABLE));
-    tablePerms.put(RootTable.ID, Collections.singleton(TablePermission.ALTER_TABLE));
-
```
Comment (ctubbsii) [inline server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKAuthorizor.java]: It looks like these were moved to ZKPermHandler, that's why it's not needed here.

**Pull request merged**